.storybook/recipes/CalendarCard/CalendarCard.module.css

```diff
@@ -0,0 +1,26 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #CALENDARCARD
+\*------------------------------------*/
+
+/**
+ * 1) 
+ */
+.calendar-card {
```

Comment:
In figma I see that the card has a background color of `Background/neutral-default-hover`. Is there a specific reason why you left that out?